data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3839 @@
+	description `Mine 100 tons of silicon from the asteroid field in Wah Yoot and 15 tons of silver from the asteroid field in <waypoints> and then bring them back to <destination> for <payment>.`
+	source "Darkcloak"
+	waypoint "Ehma Ti"
+	to offer
+		has "Unfettered: Alphas 2: Prove Yourself: done"
+	on offer
+		conversation
+			`You have never seen so much business in Unfettered space. It looks like a major undertaking is about to happen, and you will probably be a part of it. Taer-Fi is spreading a message for all Unfettered to hear. "The time has come, brothers and sisters! Now all that is needed is to spread the word, and our brethren from Hai space will join us in glorious victory! But let us not be reckless. In order to win we will need ships, and in order to build ships we will need materials. That is why we should all start gathering materials here."`
+			choice
+				`	"What kind of materials do you need?"`
+				`	(Leave. I'm not interested.)`
+					decline
+			`	She notices you and responds to your question explicitly, speaking to all that are present, "Do not bother with lead, friends, it is useless for we are not a barbaric people. No, we need a lot of silver and silicon. Also do not forget we are selling asteroid scanners in the outfitter, should you require any."`
+			choice
+				`	(I know how and where to mine asteroids. Better get going.)`
+					accept
+				`	"How do I even collect minerals from asteroids?"`
+					goto asteroidtuto
+				`	"Where can I get those resources from?"`
+					goto where
+				`	(Leave. I don't have time to start looking for those.)`
+					decline
+			label asteroidtuto
+			`	It looks like she is done speaking to the others, so now she comes to you and says, "Hello <first>. It's been a while since I last saw you. Heard you took care of that Auxiliary and brought back quite the treasure; great work! Now, back to your question: you've never mined asteroids? I recommend you install an asteroid scanner so your ship can target them automatically. Just try to keep your ship stable around one and shoot it as you would any other ships. Keep in mind they have no shields, and you might not want to waste missiles on them, since they don't shoot back."`
+			choice
+				`	"I have no more questions. See you soon."`
+					accept
+				`	"Where can I get those resources from?"`
+					goto where2
+				`	"Sorry, but I'm no miner."`
+					decline
+			label where
+			`	It looks like she is done speaking to the others, now she comes to you and says, "Hi <first>, it has been a while since I last saw you. Heard you took care of that Auxiliary and brought back quite the treasure, great work! Now back about your question, it is quite simple, silicon can be found right over our heads and silver in Ehma Ti. The silicon asteroid look similar to copper but they are bigger, but you can also just differentiate them with an asteroid scanner. Silver is easy to recognize with it's very small size, and natural resistance."`
```

Comment:
```suggestion
			`	It looks like she is done speaking to the others, now she comes to you and says, "Hi <first>, it has been a while since I last saw you. Heard you took care of that Auxiliary and brought back quite the treasure, great work! Now back about your question: it's quite simple: silicon can be found right over our heads, and silver in Ehma Ti. The silicon asteroid looks similar to copper but is bigger. You can also just differentiate them with an asteroid scanner. Silver is easy to recognize with its very small size and natural resistance."`
```